controllers/greptimedbcluster/deployers/meta.go

```diff
@@ -361,6 +361,7 @@ func buildEtcdMaintenance(etcdEndpoints []string) (clientv3.Maintenance, error)
 		DialOptions: []grpc.DialOption{grpc.WithBlock()},
 	})
 	if err != nil {
+		klog.Infof("Build etcd maintenance error:%v", err)
```

Comment:
- You should always use `klog.Errorf()` to print error message;
- Add the space between colon and `%v`, like `...error: %v`;